src/app/component-library/catalog/truncated-short-description/truncated-short-description.component.ts

```diff
@@ -0,0 +1,18 @@
+import {Component, HostBinding, Input, OnInit} from '@angular/core';
+
+@Component({
+  selector: 'truncated-short-description',
+  templateUrl: './truncated-short-description.component.html',
+})
+export class TruncatedShortDescription implements OnInit {
+  @Input() text!: string | undefined;
+  @HostBinding('class.italic') italic = false;
```

Comment:
I believe that if you host bind "class" it will overwrite any "class" directives on the component. So if you tried to add another `pt-3` on the component HTML tag with `class="pt-3"`, you couldn't, or one of the two would win. So to make things work, last time I checked we have to use something like this:


```ts
@HostBinding('class.italic') 
@HostBinding('class.whitespace-pre-line') 
@HostBinding('class.truncate-lines-5') 
cls = true;
```